bluetooth/h4_protocol.cc

```diff
@@ -151,6 +252,16 @@ void H4Protocol::OnPacketReady() {
   hci_packet_type_ = HCI_PACKET_TYPE_UNKNOWN;
 }
 
+
+typedef struct
+{
+    uint8_t         type;
+    uint8_t         event;
+    uint8_t         len;
+    uint8_t         offset;
+    uint16_t        layer_specific;
+} BT_EVENT_HDR;
+
```

Comment:
Packed struct?

Reply:
In BT stack structs for BT_HDR etc. are not packed. Without packing it was working properly. Hence did not packed the struct